cr-examples/onnx/src/test/java/oracle/code/onnx/MNISTDemo.java

```diff
@@ -147,18 +148,20 @@ public void keyPressed(KeyEvent e) {
                     scaledImage.getData().getSamples(0, 0, IMAGE_SIZE, IMAGE_SIZE, 0, imageData);
                     var imageTensor = Tensor.ofShape(new long[]{1, 1, IMAGE_SIZE, IMAGE_SIZE}, imageData);
```

Comment:
Would be nice if we could move this inside the TWR, so that we can associate the tensor memory with the arena session (and dispose everything in one go)